workflows/filtered_in_memory.md

```diff
@@ -88,18 +89,18 @@ This will genearate label file with 10000 data points with 50 distinct labels, r
 
 Label count for each unique label in the generated label file can be printed with help of following command
 ```bash
-  build/apps/utils/stats_label_data.exe --labels_file ./rand_labels_50_10K.txt --universal_label 0
+  build/apps/utils/stats_label_data --labels_file ./rand_labels_50_10K.txt --universal_label 0
 ```
 	
 Note that neither approach is designed for use with random synthetic labels, which will lead to unpredictable accuracy at search time.
 
 Now build and search the index and measure the recall using ground truth computed using bruteforce. We search for results with the filter 35.
 ```bash
-build/apps/utils/compute_groundtruth --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
+build/apps/utils/compute_groundtruth_for_filters --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
 build/apps/build_memory_index  --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./rand_labels_50_10K.txt --universal_label 0
```

Comment:
I think the path for label file is off here, It was generated in ```build/apps/utils``` and executed ```build/apps```  in so the label file path needs to be ```./utils/<label_file_name>``` for accuracy

Reply:
```build/apps/build_memory_index  --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./utils/rand_labels_50_10K.txt --universal_label 0```

Reply:
But I tested it and the label file is indeed generated in the root directory

<img width="1169" alt="image" src="https://github.com/microsoft/DiskANN/assets/83146518/f5d56494-d333-465f-9b3f-4beeb37bdc49">